packages/format-library/src/language/index.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * WordPress dependencies
+ */
+import { RichTextToolbarButton } from '@wordpress/block-editor';
+import {
+	TextControl,
+	SelectControl,
+	Button,
+	Popover,
+} from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { applyFormat, removeFormat, useAnchor } from '@wordpress/rich-text';
+import { translation } from '@wordpress/icons';
+
+const name = 'core/language';
+const title = __( 'Language' );
+
+const Edit = ( props ) => {
+	const { contentRef, isActive, onChange, value } = props;
+	const anchorRef = useAnchor( {
+		editableContentElement: contentRef.current,
+		language,
```

Comment:
```suggestion
		settings: language,
```

I think this should be a setting argument.

Two additional nitpicks:

* Can you move the `language` format definition before the component? It makes it easier to scan, IMO.
* Returned value of `useAnchor` isn't precisely a React ref. Maybe we should use `popoverAnchor` instead of `anchorRef`? It's still vague but doesn't imply a return type to be a `ref`.

Reply:
> Can you move the `language` format definition before the component? It makes it easier to scan, IMO.

Done 👍 

> Returned value of `useAnchor` isn't precisely a React ref. Maybe we should use `popoverAnchor` instead of `anchorRef`? It's still vague but doesn't imply a return type to be a `ref`.

Good point. Done 👍

Reply:
Sorry forgot that `const` declarations aren't hoisted - https://developer.mozilla.org/en-US/docs/Glossary/Hoisting.

If you swap `const Edit = ( props ) =>` with `function Edit( props )`, then it should work. Lately, I started preferring the latter for React component definitions. But it's just a personal taste.

Anyways, it's just a tiny nitpick; feel free to leave it as it is 👍